knexfile.js

```diff
@@ -7,7 +7,8 @@ const {
   DB_USER,
   DB_PASSWORD,
   DB_HOST,
-  DB_PORT
+  DB_PORT,
+  DATABASE2
```

Comment:
What is DATABASE2?

Reply:
Its supposed to be the testing database, I'm not sure if I should do TESTDB and TESTDB